examples/data_collect/main.cpp

```diff
@@ -138,15 +138,15 @@ int main(int argc, char *argv[]) {
 
     result = command.checkValue(command_map, arg_error);
     if (result != 0) {
-        LOG(ERROR) << "Argument: " << arg_error
+        LOG(ERROR) << "Argument: " << command_map[arg_error].long_option
                    << " doesn't have assigned or default value!";
         LOG(INFO) << kUsagePublic;
         return -1;
     }
 
     result = command.checkMandatoryArguments(command_map, arg_error);
     if (result != 0) {
-        std::string argName = (arg_error == "-config") ? "CONFIG" : arg_error;
+        std::string argName = (arg_error == "-config") ? "CONFIG" : command_map[arg_error].long_option;
```

Comment:
I don't want the dash(-) to be printed. It should print 'CONFIG' and not '-CONFIG' thus matching the description of the help menu.